scripts/feed2maildir

```diff
@@ -40,7 +43,11 @@ def main():
     else:
         maildir = config['maildir']
 
+    if args['strip_program']:
+        args['s'] = True
```

Comment:
Rather than modifying the `args`, I'd rather just introduce a `strip` variable and set that to `args['s'] or args['strip_program']`.